src/ClientData/include/ClientData/TracepointEventInfo.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_TRACEPOINT_EVENT_INFO_H_
+#define CLIENT_DATA_TRACEPOINT_EVENT_INFO_H_
+
+#include <cstdint>
+
+namespace orbit_client_data {
+
+// This class is used on the client to represent a tracepoint event sample on a certain thread at a
+// certain timestamp. The actual callstack is referenced by the tracepoint id.
+// See `orbit_grpc_protos::TracePointEvent`.
+class TracepointEventInfo {
+ public:
+  TracepointEventInfo(uint32_t pid, uint32_t tid, int32_t cpu, uint64_t timestamp_ns,
```

Comment:
So I guess you can explicitly delete the default constructor here as well.

Reply:
Done.